### ISSUE TITLE
make data/jaws command fails

### ISSUE THREAD

**Issue by cderinbogaz** [make-me/make-me#100]
Hello,

Finally I managed to install make-me by downgrading Osx to 10.8, but having problems with make command,

I have connected the Replicator 2 to usb and there is an installed copy of Makerware exists.

This is the error I got;

=> Loading submodules
git submodule sync
Synchronizing submodule url for 'vendor/Miracle-Grue'
Synchronizing submodule url for 'vendor/s3g'
Synchronizing submodule url for 'vendor/stltwalker'
git submodule update --init --recursive
=> Building deps
make -C vendor
Warning: Your Xcode (4.5.2) is outdated
Please update to Xcode 5.0.1.
Xcode can be updated from the App Store.
Warning: avrdude-6.0.1 already installed
Warning: Your Xcode (4.5.2) is outdated
Please update to Xcode 5.0.1.
Xcode can be updated from the App Store.
Warning: scons-2.3.0 already installed
make grue
make[2]: Nothing to be done for `grue'.
make s3g
make[2]: Nothing to be done for`s3g'.
make -C stltwalker
make[2]: Nothing to be done for `all'.
Building gcode: At[data/jaws.gcode] In[data/jaws.stl]
Argument #0 name startGcode is #/dev/null
Argument #1 name endGcode is #/dev/null
Argument #2 name config is #/Users/boyutla/make-me/config/grue-default.config
Argument #3 name outFilename is #/Users/boyutla/make-me/data/jaws.gcode
:outlines [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
Loop Processing [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
insets [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
2013-11-24 17:19:37
spurs [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
2013-11-24 17:19:37
flat surfaces [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
roofing [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
flooring [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
infills [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
Path generation [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
gcode [10%]  [20%]  [30%]  [40%]  [50%]  [60%]  [70%]  [80%]  [90%]  [100%]  [110%] 2013-11-24 17:19:37
Printing
(                                       \
        /Users/boyutla/make-me/bin/print_gcode -m "The Replicator 2" -p /dev/tty.usbmodemfa131 -f /Users/boyutla/make-me/data/jaws.gcode &           \
        echo $! > /Users/boyutla/make-me/tmp/print.pid;   \
        wait`cat /Users/boyutla/make-me/tmp/print.pid` && \
        rm /Users/boyutla/make-me/tmp/print.pid;           \
    )
Traceback (most recent call last):
  File "/Users/boyutla/make-me/bin/../bin/print_gcode.py", line 40, in <module>
    start_gcode = assembler.assemble_start_sequence(start)
  File "/Users/boyutla/make-me/vendor/s3g/makerbot_driver/GcodeAssembler.py", line 157, in assemble_start_sequence
    recipe, template_name, order)
  File "/Users/boyutla/make-me/vendor/s3g/makerbot_driver/GcodeAssembler.py", line 185, in assemble_sequence_from_recipe
    template = self.machine_profile.values[template_name]
AttributeError: 'NoneType' object has no attribute 'values'
make: **\* [data/jaws] Error 1
rm data/jaws.gcode
